static/js/settings.js

```diff
@@ -122,6 +124,29 @@ var SettingsScreen = {
       console.error('Error getting floorplan experiment setting ' + e);
     });
 
+    window.API.getExperimentSetting('speech')
```

Comment:
nitpick: this splits up the getExperimentSetting('floorplan') block above from the logically related `this.floorplanExperimentCheckbox.addEventListener` block below

Reply:
Thanks! Just reordered them!